client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/contact-list/utils.ts

```diff
@@ -0,0 +1,37 @@
+import {
+	AllowedMonitorContactActions,
+	StateMonitorSettingsEmail,
+	StateMonitorSettingsSMS,
+} from '../../sites-overview/types';
+
+export const getContactItemValue = (
+	type: 'email' | 'phone',
+	item: StateMonitorSettingsEmail | StateMonitorSettingsSMS
+) => {
+	if ( type === 'email' ) {
```

Comment:
Can we also check if the `type` is phone here? Let's not have a default one considering we will have other modes of notifications.